srcpkgs/cmake-bootstrap/template

```diff
@@ -0,0 +1,52 @@
+# Template file for 'cmake-bootstrap'
+pkgname=cmake-bootstrap
+version=3.18.2
+revision=1
+wrksrc=cmake-$version
+bootstrap=yes
+build_style=configure
+configure_args="--prefix=/usr --datadir=share/$pkgname --bindir=bin
+ --system-libs --no-system-curl --no-system-jsoncpp --no-system-nghttp2
+ --no-system-librhash --no-system-libuv --no-system-expat --no-qt-gui
+ ${XBPS_MAKEJOBS:+--parallel=$XBPS_MAKEJOBS}"
+make_build_args="-C Bootstrap.cmk"
+makedepends="libarchive-devel"
+short_desc="Cross MAKE (bootstrapping version)"
```

Comment:
bad short_desc, cmake is not a "cross make" or anything of the sort

Reply:
Heh, they self-claimed "cross-platform make" and I was lazy. I'll be more lazy to just write "CMake"